core/invoker/src/main/scala/whisk/core/container/HttpUtils.scala

```diff
@@ -66,20 +66,21 @@ protected[container] class HttpUtils(
      *
      * @param endpoint the path the api call relative to hostname
      * @param body the json object to post
+     * @param retry whether or not to retry on connection failure
      * @return Left(Error Message) or Right(Status Code, Response as UTF-8 String)
      */
-    def post(endpoint: String, body: JsObject): Either[ContainerConnectionError, ContainerResponse] = {
+    def post(endpoint: String, body: JsObject, retry: Boolean = true): Either[ContainerConnectionError, ContainerResponse] = {
```

Comment:
I understand the other two since they are private, but for this one won't all existing callers need to be found and updated to explicitly include the new parameter?